suite-native/module-accounts/src/components/TokenAccountDetailScreenHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Badge, Box, Text } from '@suite-native/atoms';
+import { ScreenHeader } from '@suite-native/navigation';
+import { AccountsRootState, selectAccountLabel } from '@suite-common/wallet-core';
+
+type TokenAccountDetailScreenHeaderProps = {
+    accountKey: string;
+    tokenName?: string;
```

Comment:
Why it is optional?

Reply:
fixed in [b5f38cd](https://github.com/trezor/trezor-suite/pull/8136/commits/b5f38cd80563e8ae8af9eba76371deb152f7c6bf)